snapcraft/internal/meta/hooks.py

```diff
@@ -14,20 +14,32 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+from collections import OrderedDict
```

Comment:
I would prefer if the most specific import (`from ...`) came after the broader import of "re"

Reply:
Fixed!